x-pack/plugins/apm/server/lib/metrics/get_cpu_chart_data/__tests__/__snapshots__/fetcher.test.ts.snap

```diff
@@ -71,7 +71,7 @@ Array [
               },
               Object {
                 "term": Object {
-                  "processor.name": "metric",
+                  "processor.event": "metric",
```

Comment:
For consistency we should always query `"processor.event` - never `"processor.name`

Reply:
good catch